integration/airflow/openlineage/airflow/extractors/manager.py

```diff
@@ -56,14 +60,23 @@ def extract_metadata(
                         self.extract_inlets_and_outlets(task_metadata, inlets, outlets)
 
                     return task_metadata
-
+            except ImportError as e:
+                if e.name is not None and e.name.startswith("airflow.providers.openlineage"):
+                    self.log.warning(
+                        "Failed to import Airflow OpenLineage provider. "
+                        "If you're using Airflow 2.7+ you should probably run "
```

Comment:
"probably"?

Reply:
I agree, this might produce some ambiguity. "please run"?